src/controllers/utils/advanced-search-validation.ts

```diff
@@ -0,0 +1,94 @@
+import { check, param } from "express-validator";
```

Comment:
Is this `param` used also. I can see `error.param` but i'm not sure whether that is the same thing?

Reply:
Unused import removed